1904-12-28.xml

```diff
@@ -0,0 +1,6056 @@
+                        permanency of their posts, rather than of the rights of the accused persons
+                        who are brought before them. Such con iderations might possibly operate on
+                        the minds Of the Judges of First Instance, having regard to the
+                        unceremonious way in which their tenure of office has sometimes been
+                        treated, and also to the criticisms to which their judgments are abjected by
+                        the Judicial Adviser in his Rep its as well as by the "Comité de
+                        Surveillance." The other matter which is exercising na ve opinion, as
+                        expressed by their newspapers, the revocation of the power of appeal, which
+                        abrogates the rights and privileges hitherto enjoyed by persons accused of
+                        crime in this country. Upon this point we must admit that a reactionary
+                        tendency is observable, seeing that in all other countries the power of
+                        appeal is being considerably widened. We cannot of course foreshadow with
+                        any degree of certainty what will be the exact methods of procedure, but one
+                        thing we most earnestly hope, and that is that no discretionary powers, as
+                        regards appeal, will be conferred on the judges, for, one of the wisést who
+                        er graced the English Bench in the last century held it as his opinion that
+                        the co ferment of such a power placed the judge in most invidious position,
+                        for no judge would ser. be likely to grant an appeal when he think that he
+                        is right, and he is generally wrong when he thinks himself most right. The
+                        native Press asserts that the Legislative Council as a body is dead against
+                        the new proposals, but we are afraid that if such be the case, that
+                        consideration is hardly likely to carry very great' weight.</p>
+                </div>
+                <div type="advert" xml:id="deg-ad-spc01">
+                    <head>SPIRO CONDOLÉON.</head>
+                    <p>SMYRNA TURKEY-CARPETS.</p>
+                    <p>Made entirely of wool and vegetable colours</p>
+                    <p>THE BEST CARPETS IN THE WORLD.</p>
+                    <p>SPIRO BUILDINGS, Sharia Kasr-el-Nile, Cairo</p>
+                    <p><measure type="indexNo">29501-7-1-905</measure></p>
+                </div>
+            </div>
+            <pb n="3"/>
+            <div type="page" n="3">
+                <cols n="6"/>
+                <cb n="1"/>
+                <div type="section" feature="wire">
+                    <div type="subsection">
+
+                        <head>TO-DAY'S TELEGRAMS.</head>
+                        <div type="cable">
```

Comment:
following three lines are head, not byline (which describes an author), of the first item in the section. Don't put them inside a cable div.